src/ebonite/ext/flask/prebuild/Dockerfile.j2

```diff
@@ -0,0 +1,25 @@
+FROM python:{{ python_version }}-slim
+
+WORKDIR /app
+
+RUN true \
+    && apt update \
+    && apt install -y \
+        bash \
+        nginx \
+        supervisor \
+        gcc \
+    && pip install uwsgi==2.0.18 \
```

Comment:
bump version?

Reply:
uWSGI 2.0.18 is the latest version in PyPi. What do you mean by "bump version"?

Reply:
ok, just thought that 2.0.18 refers to the release year (now that im writing it it actually sounds stupid)